backend/routes_reports.py

```
@@ -1,6 +1,27 @@
 from flask import request, jsonify, session
-from models import db, Tool, User, Checkout, AuditLog
 from datetime import datetime, timedelta
+from models import db, Tool, User, Checkout, AuditLog
```

Ruff (0.11.9)

`models.AuditLog` imported but unused

Remove unused import: `models.AuditLog`

(F401)